src/pages/home/hero-area/hero-area.jsx

```diff
@@ -1,25 +1,52 @@
-import React, { useState } from "react";
-import PropTypes from "prop-types";
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import _ from 'lodash';
```

Comment:
Better to avoid importing all lodash to use only one function. There is some techniques how to do it. But optimization isn't the goal of the task, so it is up to you.